Systems/c172p-engine.xml

```diff
@@ -62,7 +62,100 @@
             </function>
             <output>/fdm/jsbsim/propulsion/engine[1]/air-intake-impedance-factor</output>
         </fcs_function>
-        
+
+    </channel>
+
+    <channel name="damage">
+        <fcs_function name="/damage/rpm-over-redline">
+            <function>
+                <sum>
+                    <property>/engines/active-engine/rpm</property>
+                    <value>-2750</value>
+                </sum>
+            </function>
+            <clipto>
+                <min>0</min>
+                <max>10000</max>
+            </clipto>
+        </fcs_function>
+
+        <fcs_function name="/damage/damage-level">
+              <function>
+                <ifthen>
+                    <eq>
+                        <p>/engines/active-engine/damage_allowed</p>
+                        <v>1</v>
+                    </eq>
+                    <sum>
+                        <property>/damage/damage-level</property>
+                        <product>
+                           <table>
+                                <independentVar lookup="row">/damage/rpm-over-redline</independentVar>
+                                <tableData>
+                                    0    0
+                                  100    1
+                                  200    4
+                                  300    9
+                                  400   16
+                                  500   25
+                                  600   36
+                                  700   49
+                                  800   64
+                                  900   81
+                                 1000  100
+                                 1100  121
+                                 1200  144
+                                 1300  169
+                                 1400  196
+                                 1500  225
+                                 9999  300
```

Comment:
This row is a bit odd. Why 9999?

Reply:
I was not sure if it would just cut off at 1,500 and 225 damage levels per second, or if it would continue being interpolated past 1,500 rpm over redline.

Reply:
It should cut off at 1500.